ext/puma_http11/org/jruby/puma/MiniSSL.java

```diff
@@ -143,6 +161,7 @@ public IRubyObject initialize(ThreadContext threadContext, IRubyObject miniSSLCo
     KeyStore ks = KeyStore.getInstance(KeyStore.getDefaultType());
     KeyStore ts = KeyStore.getInstance(KeyStore.getDefaultType());
 
+    String alias = miniSSLContext.callMethod(threadContext, "alias").convertToString().asJavaString();
```

Comment:
Consumers must now provide the _alias_ of the certificate name within the KeyStore.
So that we know which PrivateKey to pull out to setup as the TLS for the server.

```
keytool -list -v -keystore examples/puma/keystore.jks -storepass blahblah                                                    
Keystore type: jks
Keystore provider: SUN

Your keystore contains 1 entry

Alias name: mydomain
Creation date: Aug 23, 2012
Entry type: PrivateKeyEntry
```

NB* I would have preferred to avoid KeyStore entirely and work directly with PEM files; but I find this to be less disruptive.